pom.xml

```diff
@@ -69,29 +69,41 @@
 
   <repositories>
     <repository>
-      <id>sonatype.release</id>
-      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
+      <id>sonatype</id>
+      <url>https://oss.sonatype.org/content/groups/public</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
     </repository>
     <repository>
       <id>sonatype-snapshots</id>
       <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+      <releases>
+        <enabled>false</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
     </repository>
   </repositories>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>6.1.1</cdap.version>
-    <hadoop.version>2.3.0</hadoop.version>
+    <cdap.version>6.11.0</cdap.version>
+    <hadoop.version>3.3.6</hadoop.version>
     <junit.version>4.11</junit.version>
-    <hydrator.version>2.3.0-SNAPSHOT</hydrator.version>
-    <es.version>7.6.2</es.version>
-    <es-hadoop.version>7.6.2</es-hadoop.version>
+    <hydrator.version>2.13.0</hydrator.version>
+    <es.version>8.16.6</es.version>
```

Comment:
Latest elastic search clusters are getting created as 9.0.0 on cloud. we can bump up this to 9.0.0 to work properly with newer versions.